### PR TITLE
Update README and make the assets and samples split

### PR DESCRIPTION
**Pull request by splovyt**
I have updated the README with the VCPI urls and the tutorial button. In addition, I have separated the samples and assets.

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Sample Details
+
+## Test Examples (samples/test-examples.tsv)
+
+This tab-separated-values file contains a fraction of the [IBM Claim Stance Dataset](http://www.research.ibm.com/haifa/dept/vst/debating_data.shtml) ([CC-BY-SA](http://creativecommons.org/licenses/by-sa/3.0/)) not used for finetuning. In the first column, the claim is listed. In the second column, the corresponding sentiment ('pos' or 'neg') is listed. Claims in this file may be used to try out and benchmark the performance of this model.
```
Comment (bdwyer2) [inline samples/README.md]:
```suggestion
This tab-separated-values file contains a fraction of the [IBM Claim Stance Dataset](http://www.research.ibm.com/haifa/dept/vst/debating_data.shtml#Claim%20Stance) ([CC-BY-SA](http://creativecommons.org/licenses/by-sa/3.0/)) not used for fine-tuning. In the first column, the claim is listed. In the second column, the corresponding sentiment (`pos` or `neg`) is listed. Claims in this file may be used to try out and benchmark the performance of this model.
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
 [![Build Status](https://travis-ci.com/IBM/MAX-Text-Sentiment-Classifier.svg?branch=master)](https://travis-ci.com/IBM/MAX-Text-Sentiment-Classifier) [![API demo](https://img.shields.io/website/http/max-text-sentiment-classifier.max.us-south.containers.appdomain.cloud/swagger.json.svg?label=API%20demo&down_message=down&up_message=up)](http://max-text-sentiment-classifier.max.us-south.containers.appdomain.cloud/)
 
+[<img src="docs/deploy-max-to-ibm-cloud-with-kubernetes-button.png" width="400px">](http://ibm.biz/max-to-ibm-cloud-tutorial)
```
Comment (bdwyer2) [inline README.md:3]: This image doesn't seem to be showing when I preview your changes. Can you please verify the markdown is correct?

**Review by MLnick — APPROVED** [review]

File: samples/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Sample Details
+
+## Test Examples (samples/test-examples.tsv)
```
Comment (MLnick) [inline samples/README.md]: Minor nit - perhaps the following?
```
## Test Examples

The tab-separated-values file `test-examples.tsv` in the `samples` folder contains ...
```

**Review by MLnick — DISMISSED** [review]

File: samples/README.md (hunk 3 above)
Comment (splovyt) [reply samples/README.md]: Done, thanks!

**Review by MLnick — APPROVED** [review]

**Review by bdwyer2 — APPROVED** [review]

**Pull request merged**